pkgs/development/python-modules/jinja2/default.nix

```diff
@@ -54,16 +58,34 @@ buildPythonPackage rec {
     "-p no:warnings"
   ];
 
-  meta = with lib; {
-    homepage = "https://jinja.palletsprojects.com/";
-    description = "Stand-alone template engine";
-    license = licenses.bsd3;
-    longDescription = ''
-      Jinja is a fast, expressive, extensible templating engine. Special
-      placeholders in the template allow writing code similar to Python
-      syntax. Then the template is passed data to render the final document.
-      an optional sandboxed environment.
-    '';
-    maintainers = with maintainers; [ pierron ];
+  passthru = {
+    doc = stdenv.mkDerivation {
+      # Forge look and feel of multi-output derivation as best as we can.
+      #
+      # Using 'outputs = [ "doc" ];' breaks a lot of assumptions.
+      name = "${pname}-${version}-doc";
+      inherit src pname version;
+
+      patches = [
+        # Fix import of "sphinxcontrib-log-cabinet"
+        ./patches/import-order.patch
+      ];
+
+      postInstallSphinx = ''
+        mv $out/share/doc/* $out/share/doc/python$pythonVersion-$pname-$version
+      '';
+
+      nativeBuildInputs = [
+        sphinxHook
+        sphinxcontrib-log-cabinet
+        pallets-sphinx-themes
+        sphinx-issues
+      ];
+
+      inherit (python) pythonVersion;
+      inherit meta;
+    };
   };
+
+  inherit meta;
```

Comment:
I would rather keep meta here where people expect it and rely on the rec to get it into passthru. We could do that with the other variables, too, to clean out the let in.

Reply:
Good point. Got rid of let-in, moved everything into `passthru.doc`. Patch became smaller. No rebuilds since previous revision.